torch/_dynamo/output_graph.py

```diff
@@ -2198,6 +2150,23 @@ def lift_tracked_freevar_to_input(self, proxy):
         assert (
             self.parent is not None
         ), "lift_tracked_freevar_to_input should not be called on root SubgraphTracer"
+
+        example_value = proxy.node.meta["example_value"]
+
+        # For SymInt with basic symbols, it's possible that the basic symbols are lifted twice.
+        # For example, the basica symbols may have already been lifted for current subgraph when
+        # we automatically lift basic symbols in the sizes/strides of a tensor t.
+        # Suppose parent graph calls sz = t.size()[0], it creates
+        # a proxy in parent and the subgraph accesses sz via closure. sz's proxy is not tracked
+        # in current sub-tracer so we may lift the same symbol twice.
+        # To avoid this case, we always call _lift_symbols_in_symint to handle basic symbols.
+        #
+        # For SymInt with compound expresssion, we allow them to be lifted as subgraph's input.
```

Comment:
Hold on, we're saying that s0*s1 can be a graph input, or are we lifting both s0 and s1 to graph inputs when we see s0*s1?